.github/resources/apiserver/root-ca-configmap.yaml

```diff
@@ -0,0 +1,40 @@
+kind: ConfigMap
```

Comment:
naming this folder `apiserver` for this file is a bit confusing, we don't deploy apiserver, the DSPO does. Let's name the folder `tls`. 

Also, let's leave a comment for the maintainer on what this rootCA is being used for.

Reply:
can you actually create the following instead: 

```yaml
├── resources
│   ├── external-pre-reqs
│   │   ├── kustomization.yaml
│   │   ├── mariadb-secret.yaml               
│   │   ├── minio-secret.yaml
│   │   └── root-ca.yaml
```

* mariadb-secret.yaml has the same contents as [this](https://github.com/opendatahub-io/data-science-pipelines-operator/blob/main/.github/resources/mariadb/secret.yaml)
*  minio-secret.yaml has the same contents as [this](https://github.com/opendatahub-io/data-science-pipelines-operator/blob/main/.github/resources/minio/secret.yaml)


then get rid of [this](https://github.com/DharmitD/data-science-pipelines-operator/blob/external-tls/.github/workflows/kind-integration.yml#L117-L130). And deploy them all together via `kustomize build` 

in `kustomization.yaml` above, link the folders to the [mariadb](https://github.com/DharmitD/data-science-pipelines-operator/tree/external-tls/.github/resources/mariadb) and [minio](https://github.com/DharmitD/data-science-pipelines-operator/tree/external-tls/.github/resources/minio) for reference